docs/hw/hookup.md

```diff
@@ -1,87 +1,18 @@
-# iRobot® Create® 3 Hookup Guide
+# iRobot® Create® 3 Hardware Hookup Overview
```

Comment:
this `hookup.md` page is now not listed anymore in the website navigation tree

Reply:
Yes, that is intentional as some sites and old links point to the hw/hookup url. By leaving it not listed in the yml nav tree it allows old links to still get to the page and see the newer subpage redirects.

I believe this addresses that potential issue(s) that @shamlian had brought up of the structure change.

If you are building it locally you can test out how it would work with the otherwise "broken link" by going to: `http://127.0.0.1:8000/hw/hookup`

Reply:
Ok sounds good to me.
Just wanted to make sure that it was intentional.

Reply:
Yes, this addresses my concern very nicely (thanks @bperseghetti)!